Explorer/Assets/DCL/Navmap/FloatingPanelController.cs

```diff
@@ -144,14 +156,17 @@ private void SetEmptyParcelInfo(Vector2Int parcel)
             view.upvotes.text = "-";
             view.parcelsCount.text = "1";
             placeImageController.SetVisible(false);
+            mapPinPlaceImageController.SetVisible(false);
 
             ResetCategories();
         }
 
         private void SetFloatingPanelInfo(PlacesData.PlaceInfo placeInfo)
         {
             placeImageController.SetVisible(true);
+            mapPinPlaceImageController.SetVisible(true);
             placeImageController.RequestImage(placeInfo.image);
+            mapPinPlaceImageController.RequestImage(placeInfo.image);

```

Comment:
Is it ok to set both the placeImageController and the mapPinPlaceImageControllers to visible and request the image for both?